apps/parsley/vite.config.ts

```diff
@@ -12,114 +12,119 @@ import path from "path";
 import { generateBaseHTTPSViteServerConfig } from "@evg-ui/vite-utils";
 import injectVariablesInHTML from "./config/injectVariablesInHTML";
 
-// Remove when https://github.com/cypress-io/cypress/issues/25397 is resolved.
-dns.setDefaultResultOrder("ipv4first");
+let finalConfig: Record<string, any> = {};
```

Comment:
I didn't actually test to see if this works, but it could help with readability to write it as
```suggestion
const useProjectConfig = process.env.VITE_SCRIPT_MODE !== "1";

const finalConfig = useProjectConfig ? getProjectConfig() : {};

export default finalConfig;
```

where `getProjectConfig()` contains the project config logic.